controllers/yawol-controller/loadbalancer/loadbalancer_controller.go

```diff
@@ -1117,6 +1118,24 @@ func (r *Reconciler) deleteFips(
 
 	var requeue = false
 
+	// skip deletion and release status when annotated
+	if keep, err := strconv.ParseBool(lb.GetAnnotations()[yawolv1beta1.LoadBalancerKeepFloatingIP]); err != nil && keep {
```

Comment:
needs to be:
```
if keep, err := strconv.ParseBool(lb.GetAnnotations()[yawolv1beta1.LoadBalancerKeepFloatingIP]); err == nil && keep {
```
or just ignore the error (is always false if a error is happen)
```
if keep, _ := strconv.ParseBool(lb.GetAnnotations()[yawolv1beta1.LoadBalancerKeepFloatingIP]); keep {
```

Reply:
done